pkg/diagnostics/network/objects.go

```diff
@@ -64,7 +66,7 @@ func GetNetworkDiagnosticsPod(command []string, podName, nodeName string) *kapi.
 							ReadOnly:  true,
 						},
 					},
-					Command: command,
+					Command: []string{"sh", "-c", cmd},
```

Comment:
you could just do `[]string{"openshift-network-debug", util.NetworkDiagContainerMountPath, command}` and then in the script set `cmd="$2"` rather than `shift; cmd="$@"`